### PR TITLE
Bottom Alert Modals

### PR DESCRIPTION
**Pull request by ItzNotABug**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,6 +73,11 @@
         "vite": "^5.4.0",
         "vitest": "^1.6.0"
     },
+    "pnpm": {
+        "overrides": {
+            "rollup": "^4.22.4"
+        }
+    },
```
Comment (TorstenDittmann) [inline package.json:80]: why is this needed?

File: tests/e2e/journeys/upgrade-free-tier.spec.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ test('upgrade - free tier', async ({ page }) => {
     await registerUserStep(page);
     await createFreeProject(page);
     await test.step('upgrade project', async () => {
-        await page.getByRole('link', { name: 'upgrade' }).click();
+        await page.getByRole('link', { name: 'Upgrade', exact: true }).click();
```
Comment (TorstenDittmann) [inline tests/e2e/journeys/upgrade-free-tier.spec.ts:10]: how come this needs to be exact now?

File: package.json (hunk 1 above)
Comment (ItzNotABug) [reply package.json:80]: Pnpm audit fails due to a vulnerability in rollup which is a dependency on vite. vite hasn't released a fix yet so this takes care of the audit for the time being.

File: tests/e2e/journeys/upgrade-free-tier.spec.ts (hunk 2 above)
Comment (ItzNotABug) [reply tests/e2e/journeys/upgrade-free-tier.spec.ts:10]: If there is any 'update' text in an anchor link with higher z-index than the main content, e2e seems to fail. Prior commits x tests can be seen where this happens. Assuming the e2e calls the Update in the header, this should fix it.

File: src/lib/components/bottomModalAlert.svelte [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+                return (
+                    organizationPlan === BillingPlan.FREE || organizationPlan === BillingPlan.PRO
+                );
+        }
+    }
+
+    onMount(() => {
+        addBottomModalAlerts();
+    });
+
```
Comment (TorstenDittmann) [inline src/lib/components/bottomModalAlert.svelte:66]:
```suggestion
```

**Review by TorstenDittmann — APPROVED** [review]
